guides/source/index.html.erb

```diff
@@ -10,7 +10,7 @@
   <dl>
     <dt></dt>
     <% unless @edge -%>
-      <dd class="interstitial info kindle">Rails Guides are also available for <%= link_to 'Kindle', @epub %>.</dd>
+      <dd class="interstitial info kindle"><p>Rails Guides are also available for <%= link_to 'Kindle', @epub %>.</p></dd>
```

Comment:
why is this one wrapped with `p` but not the others?

Reply:
Hello @MatheusRich 
Please check the screenshot attached, since we are applying display flex and align item center, this was breaking because the `dd` element contains 3 element under it and it was breaking the style. 
and if you can check in the codebase whereever we are applying the `interstitial` class we are adding the inner content in the <p> tag, so just for consistency as well to make sure nothing breaks.